consumer-features/example-hello-world/v3/example-hello-world-js-jest-pact/Makefile

```diff
@@ -0,0 +1,2 @@
+test:
```

Comment:
Would it make more sense if everything implemented a two step
make deps
make test
I think it might not add value if you always use the two normally, but I'm still trying to think what would make sense to be able to distinguish what pact version you're using where etc without making it excessively complicated, so might be cleaner if it's separate [and the runner does both for now at least]

Reply:
yes 110%